Ginger/Ginger/GeneralLib/General.cs

```diff
@@ -534,6 +534,11 @@ public static string GetTagsListAsString(ObservableList<Guid> tagsIDsList)
 
         public static bool UndoChangesInRepositoryItem(RepositoryItemBase item, bool isLocalBackup = false, bool clearBackup = true)
         {
+            if ((isLocalBackup && item.IsLocalBackupExist == false) || (isLocalBackup == false && item.IsBackupExist == false))
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)